spire/templates/apps/dovetail-metrics-export.yml

```diff
@@ -239,3 +239,36 @@ Resources:
       FunctionName: !Ref ExportFunction
       Principal: events.amazonaws.com
       SourceArn: !GetAtt TedNetworkExport.Arn
+
+  BoostrIntegration:
+    Type: AWS::Events::Rule
+    Condition: EnableProductionRules
+    Properties:
+      Description: Boostr integration impression export
+      RoleArn: !GetAtt RulesRole.Arn
+      ScheduleExpression: cron(15 0 * * ? *) # 00:15 UTC every day
+      State: ENABLED
+      Targets:
+        - Arn: !GetAtt ExportFunction.Arn
+          Id: !Sub "${RootStackName}.Export.BoostrIntegration"
+          Input: |-
+            {
+              "IntegrationIds": [1],
+              "CompressionType": "NONE",
+              "DestinationFormat": "CSV",
+              "ObjectPrefix": "boostr/",
```

Comment:
In my original doc, I said the format would be:

```
prod/soundrise/boostr/2023-06-12T00:00:00.000Z/FILE_SEQ_ID.csv.gz
```

I guess my only question is - will the default format potentially result in multiple files for the same day, or overwrite?  Probably better to just overwrite, for this use?